Homework1/Sort/Sort/BubbleSort.cs

```diff
@@ -0,0 +1,43 @@
+﻿using System;
+
+namespace Sort
+{
+    public class BubbleSort
+    {
+        public void BubbleSortArray(int[] arrayOfNumbers)
+        {
+            for (int i = 0; i < arrayOfNumbers.Length - 1; i++)
+            {
+                for (int j = arrayOfNumbers.Length - 1; j > i; j--)
+                {
+                    if (arrayOfNumbers[j - 1] > arrayOfNumbers[j])
+                    {
+                        int temporary = arrayOfNumbers[j];
+                        arrayOfNumbers[j] = arrayOfNumbers[j - 1];
+                        arrayOfNumbers[j - 1] = temporary;
```

Comment:
А можно как в Python:
```suggestion
                        (arrayOfNumbers[j - 1], arrayOfNumbers[j]) = (arrayOfNumbers[j], arrayOfNumbers[j - 1]);
```